zerver/views/streams.py

```diff
@@ -454,7 +454,7 @@ def get_subscription_or_die(stream_name, user_profile):
     # type: (text_type, UserProfile) -> Subscription
     stream = get_stream(stream_name, user_profile.realm)
     if not stream:
-        raise JsonableError(_("Invalid stream %s") % (stream.name,))
```

Comment:
Hello @timabbott ,

If an invalid stream is passed, stream is a 'NoneType' object. 
Therefore modification is done here.
Is it fine?